frontend/src/views/WheresMyGeneV2/components/GeneSearchBar/components/ShareButton/utils.tsx

```diff
@@ -76,6 +76,7 @@ export const loadStateFromQueryParams = (
   params: URLSearchParams,
   selectedFilters: State["selectedFilters"],
   dispatch: Dispatch<PayloadAction<LoadStateFromURLPayload>>
+  // eslint-disable-next-line sonarjs/cognitive-complexity -- TODO(thuang): Update this after https://github.com/chanzuckerberg/single-cell-data-portal/pull/5531 is merged
```

Comment:
will refactor this after https://github.com/chanzuckerberg/single-cell-data-portal/pull/5531 is merged